sources/packages/api/src/utilities/auth-utils.spec.ts

```diff
@@ -0,0 +1,22 @@
+// Target module
```

Comment:
Keep inside test folder?

Reply:
The unit tests for the API are alongside the files that they are testing. We can change it, but it should be a change for all others also. Let's talk about it.